scripts/pre_leap.sh

```diff
@@ -19,23 +19,18 @@
 set -e -u
 set -o pipefail
 
+# Pre-flight check
+pushd /opt/rpc-openstack/rpcd/playbooks
```

Comment:
The gates are currently failing because it can't find inventory.  Try this instead, change into /opt/rpc-upgrades/playbooks.  You'll probably need to drop this file into playbooks in your commit:

https://github.com/antonym/rpc-upgrades/blob/74f58b5364de25d114ca857df237ac384c0d017a/playbooks/ansible.cfg

That will allow it to automatically detect the inventory used in rpc-o and then we can see if the gates pass.

Reply:
@Frank-ZhangXin Make sure to change into /opt/rpc-upgrades/playbooks/ instead.